src/controllers/DriverController.js

```diff
@@ -1,31 +1,93 @@
-import jwt from 'jsonwebtoken';
+// import jwt from 'jsonwebtoken';
 import Model from '../models/model';
+import assignToken from '../validations/validate';
+
+// const secretKey = process.env.SECRET_KEY;
 
-const secretKey = process.env.SECRET_KEY;
 const driverModel = new Model('drivers');
 
+/**
+ * @description - This method will handle the request for adding rigister driver
+ * to the database,and return back driver object which is a result of a promise
+ *
+ * @param {object} req - request
+ *
+ * @param {object} res - response
+ *
+ * @return {object} - it return object with a token
+ */
 export const addDriver = async (req, res) => {
   const {
     firstName, lastName, address, phone, email, password
   } = req.body;
-  const columns = 'first_name, last_name, address, phone, email, password';
+  const columns = '"firstName", "lastName", address, phone, email, password';
+  // eslint-disable-next-line max-len
   const values = `'${firstName}', '${lastName}', '${address}', '${phone}', '${email}', '${password}'`;
   try {
     const data = await driverModel.insertWithReturn(columns, values);
     const { id } = data.rows[0];
-    const token = jwt.sign({
+    const driver = {
       id,
-      email
-    }, secretKey, {
-      expiresIn: '24h'
-    });
+      email: data.rows[0].email
+    };
+    const token = assignToken(driver);
     res.status(200).json({
-      id,
-      email,
+      driver,
       token,
       message: 'Account created successfully!'
     });
   } catch (err) {
-    res.status(500).json(err);
+    res.status(500).json(
+      { message: `Internal server '${err.severity}', please re-try`, success: false }
+    );
+  }
+};
+
+/**
+ * @description - This method hadle the request for updating driver profile
+ * it return an object with "success message" or "access denied" if user is not valid
+ *
+ * @param {object} req - request
+ *
+ * @param {object} res - response
+ *
+ * @returns {object} - it return object with a message if user is valid or invalid
+ */
+export const editDriverProfile = async (req, res) => {
+  const { id } = req.user.data;
+  try {
+    const data = await driverModel.update(req.body, `WHERE id = '${id}'`);
+    if (data.rowCount === 0) {
+      return res.status(401).json(
+        { Message: 'unauthorized, Please login or register', success: false }
```

Comment:
This error message is wrong, the user is already logged in at this point, you should either send a message to tell them that the item they are trying to delete doesn't exist and the status code should be 404